drift/info_parser.py

```diff
@@ -140,6 +140,7 @@ def _group_comparisons(comparisons):
 
     # default values for method run
     grouped_comparisons = []
+    temp_grouped_comparisons_dict = {}
```

Comment:
We can't transform a dict into a list, just "transforming" it.. in our case we need to create a new list and iterate into that dict passing the new values, since dict we don't have the same output format we had in the list.